api/v1alpha/istio-manifests/1.4/authz/allow-data-producers-policy.yaml

```diff
@@ -7,6 +7,31 @@ metadata:
 spec:
   action: ALLOW
   rules:
+  - to:
+    - operation:
+      methods: ["GET,POST,PUT,DELETE"]
+      paths: ["/v1alpha/projects/$PROJECT_ID/accounts*, /v1alpha/projects/$PROJECT_ID/accounts/*"]
```

Comment:
Is a wildcard asterisk '*' is required for both the resource `/accounts*` and the sub resources `/accounts/*` or just the first option?